Deep-Drop/core/deepdrop.py

```diff
@@ -33,30 +32,26 @@ def parse_process_list(process_list):
 
     processes = namedtuple('process', 'pid, owner, name')
 
+    hostname = ''
     try:
         task_result = base64.b64decode(process_list).decode().split('\r\n')
-
         hostname = task_result[0].split('=')[1]
-
     except Exception as e:
         print(e)
 
-        pass
-
     if task_result[1] == '':
         logging.error('Process list is empty')
-
         return 'Hello'
-        
+
     for process in task_result[1:]:
         full_result = re.findall(r'^([0-9]+)\s+(\S+)\s+(.+)$', process)
 
         if full_result:
-            (pid, owner, name) = full_result[0]
-            
+            pid, owner, name = full_result[0]
```

Comment:
Not functionally different.

Reply:
Indeed. It's just a style fix to remove the unnecessary parentheses. If your personal preference is to use parentheses around tuple assignments, that's fine too.